Decipher-Framework.py

```diff
@@ -33,14 +41,36 @@ def decipher_base64():
   E for Encrypt
   D for Decrypt''')
     user_input = input('Operation: ').lower()
+
+    if user_input == 'e':
+        print(f'''{Fore.RED}Do you want to encrypt a file?
+        Y for Yes
+        N for No''')
+    else:
+        print(f'''{Fore.RED}Do you want to decrypt a file?
+        Y for Yes
+        N for No''')
+    file_choice = input('Operation: ').lower()
+
+    if file_choice == 'y':
+        file_path = input("What is the path of the file? ")
+        file_content = read_file(file_path)
+    else:
+        print('Please enter the string you would like Base64 encoded')
+        string_input = input('String: ')
+
     if user_input in options:
-        if user_input == 'e':
-            print('Please enter the string you would like Base64 encoded')
-            string_input = input('String: ')
-            print(f'\n{Fore.WHITE}our Ecrypted Text >>> {base64.b64encode(string_input.encode()).decode()}')
+        if user_input == 'e' and file_choice == 'y':
+            encoded_string = base64.b64encode(file_content.encode()).decode()
+            print(f'\n{Fore.WHITE}Your Ecrypted Text >>> {encoded_string}')
+            write_file(file_path, encoded_string)
+        elif user_input == 'd' and file_choice == 'y':
+            encoded_string = base64.b64decode(file_content.encode()).decode()
+            print(f'\n{Fore.WHITE}Your Ecrypted Text >>> {encoded_string}')
+            write_file(file_path, encoded_string)
+        elif user_input == 'e':
+            print(f'\n{Fore.WHITE}Your Ecrypted Text >>> {base64.b64encode(string_input.encode()).decode()}')
         elif user_input == 'd':
-            print('Please enter the string you would like Base64 decoded')
-            string_input = input('String: ')
```

Comment:
I Would like to keep this prompt so user has a way of knowing what to type and the availability of a prompt makes it better and readable by the user i think. What do you think?